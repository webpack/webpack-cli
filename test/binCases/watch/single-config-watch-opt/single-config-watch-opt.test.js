"use strict";

const { runWatch } = require("../../../testUtils");
test("single-config-watch-opt", () => {
	runWatch(__dirname, [
		"--entry",
		"./index.js",
		"--config",
		"./webpack.config.js",
		"--output-filename",
		"[name].js",
		"--output-chunk-filename",
		"[id].chunk.js",
		"--target",
		"async-node",
		"--watch"
	]).then(result => {
		const { stdout, stderr } = result;
		expect(stdout).toEqual(expect.anything());
		expect(stdout).toContain("");
		expect(stdout).toContain("webpack is watching the filesâ€¦");

		expect(stderr).toHaveLength(0);

		expect(stdout).toMatchSnapshot();
		return;
	});
});
