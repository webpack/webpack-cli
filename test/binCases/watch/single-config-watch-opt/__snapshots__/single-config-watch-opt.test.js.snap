// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single-config-watch-opt 1`] = `
"  Asset       Size  Chunks             Chunk Names
null.js  930 bytes       0  [emitted]  null
Entrypoint null = null.js
[0] ./index.js 0 bytes {0} [built]

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/concepts/mode/
"
`;
